boards/arduino-due/include/periph_conf.h

```diff
@@ -342,6 +343,34 @@ extern "C" {
 #define GPIO_B21_MAP        31
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define MCK_DIV_LB_MAX      (10U)
+#define PWM_NUMOF           (1U)
+#define PWM_0_EN            (1)
+#define PWM_MAX_VALUE       (0xffff)
+//#define PWM_MAX_CHANNELS    4
+
+/* Temporary PC2 -- Pin34 XIO-15 */
```

Comment:
Just a relict from early implementation where I used channel 0, and therefore had to deactive GPIO 22. Will be removed.